dev/CONTRIBUTING.md

```diff
@@ -100,87 +94,60 @@ Run `builder run check` before committing to ensure lint and tests are passing.
+
+**Support Data Accessor Props**
+
+Rather than requiring a rigid data structure, Victory components should provide accessor props for formatting whatever data the user provides into a format the component is expecting. Use this [`createAccessor` function](https://github.com/FormidableLabs/victory-core/blob/master/src/victory-util/helpers.js#L117) to build an accessor from props and use it to format data. 
+
+**Support themes**
+
+`defaultProps` should not contain any layout related props because these values may be provided by themes. Necessary layout props should be provided via [`fallbackProps`](https://github.com/FormidableLabs/victory-chart/blob/master/src/components/victory-scatter/victory-scatter.js#L11) instead, and merged with props and themes as needed.
+
+### Sensible Defaults
+
+All Victory components should render _something_ even if no props are provided. This can be accomplished with default data and other `defaultProps`. Victory components should also have sensible behavior even when the minimum of props are provided. For example, if only `data` is provided, a component like `VictoryBar` should calculate a domain from the data provided. All Victory components should also use the default `grayscale` theme so that they have sensible styling even when no styles are provided by the user.
+
+### Consistency
+
+Similar props across Victory components should have the same names and operate similarly. For example, the style prop should almost always have the shape:
+
+```
 style={{
-  parent: {
-    border: "1px solid #ccc",
-    height: 500,
-    margin: 50,
-    width: 500
-  },
   data: {
-    fill: "red",
-    opacity: 0.8
+    ...
   },
   labels: {
-    fontSize: 15,
-    padding: 20,
-    fill: "grey"
+    ...
   }
 }}
 ```
 
-- Support animating and static visualizations
-- Take an `animate` prop as an object that can be passed as props to [victory-animation](https://github.com/FormidableLabs/victory-animation)
-- Support rendering svg and g tags via a `standalone` prop (renders svg when true, g when false).
-- Not require users to write d3 or have d3 domain knowledge
-- Be accessible and usable for junior developers
-- Have an intuitive api
+When writing a new component please reference other Victory components when deciding on a set of props. When writing a component for use with VictoryChart, the API should close to a strict superset of the props described in other components like [`VictoryBar`](https://github.com/FormidableLabs/victory-chart/blob/master/src/components/victory-bar/victory-bar.js#L46)
```

Comment:
I think there's a typo (`...the API should close to a strict superset...`) and I think the wording could be more direct. Maybe:

> When writing a component for use with VictoryChart, its props should include all the props used by the other components, like VictoryBar, unless it is absolutely necessary to change them.